### ISSUE TITLE
Add example with PyMC3

### ISSUE THREAD

**Issue by twiecki** [matt-graham/mici#3]
It should be fairly easy to apply these samplers to a PyMC3 model. Our samplers are written in pure Python too so the speed should be the same. What do you think?

**Comment by matt-graham** [comment]
I agree this would definitely be something worth doing! In general it would help to have a lot more documented examples and this would be a good place to start :smile:
